plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestIcebergVendingRestCatalogConnectorSmokeTest.java

```diff
@@ -309,14 +309,6 @@ public void testDropTableWithMissingManifestListFile()
                 .hasMessageContaining("Table location should not exist");
     }
 
-    @Test
-    @Override
-    public void testDropTableWithMissingDataFile()
```

Comment:
this can only be removed once `IcebergRestCatalogBackendContainer` uses `iceberg-rest` with the fix from https://github.com/apache/iceberg/pull/9305

Reply:
I would suggest to remove this later when the `iceberg-rest` image has been released to use Iceberg 1.5.0. Then we can update the images used in `IcebergRestCatalogBackendContainer` / `EnvSinglenodeSparkIcebergRest`